lib/general-calendar/proper-of-saints.ts

```diff
@@ -442,6 +442,9 @@ export class GeneralRoman extends CalendarDef {
       precedence: Precedences.GeneralMemorial_10,
       // The Saturday, after the Solemnity of the Most Sacred Heart of Jesus
       dateDef: { dateFn: 'immaculateHeartOfMary' },
+      // If this liturgical day fall the same day of another memorial, do not
+      // replace this another memorial, but keep them both.
```

Comment:
Don’t want to nitpick on this, but here’s my suggestion. Otherwise, fix at least `fall` → `falls`. :wink:

```suggestion
      // If this liturgical day and a memorial occur on the same day,
      // do not replace the latter, but output them both.
```